assets/src/scss/layout/_navbar.scss

```diff
@@ -616,10 +632,12 @@ $navbar-default-height: 60px;
 }
 
 .country-list {
+  _-- {
+    background: $x-dark-blue;
```

Comment:
Maybe we should use the `background-color` property here instead of just `background`? It's more specific

Reply:
With background all values you'd put in `background-color` would work the same way, but you can do more things like gradients, images.

Reply:
In this case it already used `background`, in some other cases (like buttons) I changed it.